Library/Homebrew/tap.rb

```diff
@@ -349,6 +349,7 @@ def install(quiet: false, clone_target: nil, force_auto_update: nil, custom_remo
     return if clone_target
     return unless private?
     return if quiet
+    return unless `git config --get credential.helper`.empty?
```

Comment:
```suggestion
    path.cd do
      return if Utils.popen_read("git", "config", "--get", "credential.helper").present?
    end
```

Reply:
I forget to check the tap's git config... Thanks for the review!